packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -42,7 +42,7 @@ const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
 } ) => {
 	const isMobile = useViewportMatch( 'small', '<' );
 
-	if ( design.is_externally_managed && design.screenshot ) {
+	if ( design?.design_tier === 'partner' && design.screenshot ) {
```

Comment:
Do we have a constant somewhere for `partner` that we can use instead of hardcoding stings?

Reply:
[yes](https://github.com/Automattic/wp-calypso//blob/ef484b893d370fc89177563430b99c5ff774bc1f/client/components/theme-tier/constants.js#L26) but it's in `client/components` so I don't think it can be imported here.

Reply:
The audit covers more instances of `is_externally_managed` checks to refactor, so maybe I'll introduce a constant then and come back for this.

Part of me things it's just more readable to keep the redundant `is_externally_managed` prop on Design and calculate that from tiers data, rather than repeating this everywhere:

```
import {PARTNER_CONSTANT} from 'constants'
if ( design?.design_tier === PARTNER_CONSTANT) {
  ...
}
```

Reply:
> Part of me thinks it's just more readable to keep the redundant is_externally_managed prop on Design and calculate that from tiers data, rather than repeating this everywhere:

If we're not going to remove the `is_externally_managed` prop from the design object, then yeah, I agree that it's more readable to have it as is.